### DISCUSSION TITLE
Is umami.is website offline?

### DISCUSSION THREAD

**Discussion by gborgonovo** [umami-software/umami#2741] [General]
Hi there,
I'm having problems reaching out umami.is website.
Every time I'm getting

> This page isn’t workingumami.is redirected you too many times.
> [Try deleting your cookies](https://support.google.com/chrome?p=rl_error&hl=en-US).
> ERR_TOO_MANY_REDIRECTS

I tried on Chrome, Firefox and Edge too.
I deleted my cache and cookies.

Is it my problem or the website is having any problem?
Please, help me.
Thanks

**Comment by alxfsl** [comment]
I can load the main website, but it gives me the same error when I try to log in:

> The page isn’t redirecting properly
> An error occurred during a connection to cloud.umami.is.
>  This problem can sometimes be caused by disabling or refusing to accept cookies.

Tried on Firefox, Edge, and Safari.

**Comment by mikecao** [comment]
We had some DNS issues but they should be resolved now. If it's still not working, you can try clearing your browsing data. Don't worry, none of your data was lost.
